install.rst

```diff
@@ -23,475 +18,39 @@ Confirm you will be able to provision the following hardware to replicate the re
-   ``system_admin`` role, which gives you access to the System Console.
-3. From the ``town-square`` channel click the dropdown and choose the
-   ``System Console`` option
-4.  Update **Notification** > **Email** settings to setup an SMTP email service. The example below assumes AmazonSES.
-
-   -  Set *Send Email Notifications* to ``true``
-   -  Set *Require Email Verification* to ``true``
-   -  Set *Feedback Name* to ``No-Reply``
-   -  Set *Feedback Email* to ``mattermost@example.com``
-   -  Set *SMTP Username* to ``[YOUR_SMTP_USERNAME]``
-   -  Set *SMTP Password* to ``[YOUR_SMTP_PASSWORD]``
-   -  Set *SMTP Server* to ``email-smtp.us-east-1.amazonaws.com``
-   -  Set *SMTP Port* to ``465``
-   -  Set *Connection Security* to ``TLS``
-   -  Save the Settings
-
-5. Update **File** > **Storage** settings:
-
-   -  Change *Local Directory Location* from ``./data/`` to
-      ``/mattermost/data``
-
-6. Update **General** > **Logging** settings:
-
-   -  Set *Log to The Console* to ``false``
-
-7. Update **Advanced** > **Rate Limiting** settings:
-
-   -  Set *Vary By Remote Address* to ``false``
-   -  Set *Vary By HTTP Header* to ``X-Real-IP``
-
-8. Update **Advanced** > **Database** settings:
-
-   -  Set *Maximum Idle Connections* to ``20``
-   -  Set *Maximum Open Connections* to ``300``
-
-9. Feel free to modify other settings.
-10. Restart the Mattermost Service by typing:
-
-    -  ``sudo restart mattermost``
-
-Installing Mattermost Load Test
-============================================
-
-1. Download Mattermost Load Test::
-
-      git clone https://github.com/mattermost/mattermost-load-test.git
-
-2. Confirm your environment is clear (deletes ``cache.db`` if it exists)::
+Follow the "Setup NGINX Server" or "Setup NGINX with SSL" instructions in our `Production Ubuntu Install Guide <https://docs.mattermost.com/install/prod-ubuntu.html#set-up-nginx-server>`_ to setup your NGINX server.
 
-      make clean
-
-3. Build the Mattermost Load Test:: 
-
-      make install
-
-4. Run the default load test script (this takes about 40 minutes):: 
-
-      ./bin/run_example.sh
-
-The default load test script requires about 40 minutes to create all the users. This is because Mattermost users a bcrypt function to generate user passwords that is designed to be computationally intensive. 
-
-Load Test Setup
-------------------------------------------
-
-1. Change the time the kernel recycles closed connections from 60 seconds to 30 seconds. The change will need to be done on the NGINX server, Mattermost server and load tester server.
-
-   - ``sudo sysctl -w net.ipv4.tcp_fin_timeout=30``
-   - You can verify it's set correctly with
-   - ``cat /proc/sys/net/ipv4/tcp_fin_timeout``
-
-2. Change the number of usable ports for nginx and mattermost. The change will need to be done on the NGINX server, Mattermost server and load tester server.
-
-   - ``sudo sysctl -w net.ipv4.ip_local_port_range="1024 65000"``
```

Comment:
Chatted with @coreyhulen. He explained these settings. They are required for over 20000 users.